cmd/influx/query.go

```diff
@@ -42,19 +42,20 @@ func fluxQueryF(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to load query: %v", err)
 	}
 
-	orgSvc, err := newOrganizationService()
-	if err != nil {
-		return fmt.Errorf("failed to initialized organization service client: %v", err)
-	}
-
-	orgID, err := queryFlags.org.getID(orgSvc)
-	if err != nil {
-		return err
-	}
+	// TODO(jsternberg): Restore this when the influxdb client source is merged.
```

Comment:
What is "influxdb client source"?

Reply:
That's the `influxdb.from` that's incoming to the flux repository. The influx cli will also rely on it but it will add defaults from the cli.